Controller/GameController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Math_Game.Helpers;
+using Math_Game.Model;
+using Math_Game.View;
+
+namespace Math_Game.Controller
+{
+    internal class GameController
+    {
+        internal static readonly string[] operation = new[] {"A", "S", "M", "D" };
+        internal List<Game> Games = new(); 
+
+        internal void Run()
+        {
+            Display.Welcome();
+            string? name = Console.ReadLine();
+            if (String.IsNullOrEmpty(name))
+                Environment.Exit(0);
```

Comment:
```suggestion
                Environment.Exit(0);
```
Instead of existing you can tell them the name field cannot be empty :).